src/Engines/MeiliSearchEngine.php

```diff
@@ -126,6 +127,12 @@ protected function performSearch(Builder $builder, array $searchParams = [])
     {
         $meilisearch = $this->meilisearch->index($builder->index ?: $builder->model->searchableAs());
 
+        // from 0.21.0 onwards, `filters` is renamed to `filter`
+        if (version_compare($this->getInstalledMeilisearchVersion(), '0.21.0') >= 0) {
+            $searchParams['filter'] = $searchParams['filters'];
+            unset($searchParams['filters']);
+        }
```

Comment:
Instead of doing this can't we just to the following?

```suggestion
        $searchParams['filter'] = $searchParams['filters'] ?? $searchParams['filter'];
        
        unset($searchParams['filters']);
```

Reply:
I'm not sure if it's a bad thing that the `filters` key stays in place when using an older Meilisearch version?

Reply:
This is the inverse, right? This will now always send `filter` but you might still be running meilisearch >=0.20.0 and that version still expects `filters`.

Reply:
But the `filters` key will stay. Only the `filter` key will be set as well. Does it matter if a `filter` key is set on newer versions?

Reply:
Unfortunately they do a strict scheme check, so sending both `filter` and `filters` to a 0.21 will error because `filters` is not recognised.

Reply:
Right, gotcha. Thanks for clarifying that 👍